tools/pdf2txt.py

```diff
@@ -32,13 +30,7 @@ def float_or_disabled(x: str) -> FloatOrDisabled:
 def extract_text(
     files: Iterable[str] = [],
     outfile: str = '-',
-    no_laparams: bool = False,
-    all_texts: Optional[bool] = None,
-    detect_vertical: Optional[bool] = None,
-    word_margin: Optional[float] = None,
-    char_margin: Optional[float] = None,
-    line_margin: Optional[float] = None,
-    boxes_flow: Optional[FloatOrDisabled] = None,
+    laparams: Optional[LAParams] = None,
```

Comment:
Breaking change.

This method shouldn't be used directly but many people will probably do so. 

Did a quick search and seems like this is not the case: https://github.com/search?p=1&q=pdfminer.tools.pdf2txt&type=Code

So, we're fine here! :)